pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   enough_html_editor:
     git:
       url: https://github.com/linagora/enough_html_editor.git
-      ref: master
+      ref: bugfix/ios-editor-height
```

Comment:
please bump patch version and update it here

Reply:
Done